packages/commonjs/README.md

```diff
@@ -127,6 +127,32 @@ Default: `[]`
 
 Sometimes you have to leave require statements unconverted. Pass an array containing the IDs or an `id => boolean` function. Only use this option if you know what you're doing!
 
+### `dynamicRequireTargets`
+
+Type: `String|Array[String]`<br>
+Default: `[]`
+
+Some modules have dynamic `require` calls in them, or even circular dependencies (which are not handled well by EJS).  
+Including those modules as `dynamicRequireTargets` will simulate a CJS environment for them with support for dynamic and circular dependencies.  
+Note that this might hardcode some paths from your PC in the final js file, so you might want to search & replace the base path of the root js file.
```

Comment:
```suggestion

_Note: This feature may result in some paths being rendered as absolute in the final bundle. That may require replacing ...<please explain more>._
```

I'm not quite sure what this note is supposed to tell the user they might have to do. I've suggested a change to the note for clearer initial language, but we nee to explain better to users what's up.